diff --git a/boring-sys/build/main.rs b/boring-sys/build/main.rs
index ae6df99..3233c13 100644
--- a/boring-sys/build/main.rs
+++ b/boring-sys/build/main.rs
@@ -69,6 +69,15 @@ const CMAKE_PARAMS_APPLE: &[(&str, &[(&str, &str)])] = &[
             ("CMAKE_MACOSX_BUNDLE", "OFF"),
         ],
     ),
+    // Mac Catalyst
+    (
+        "aarch64-apple-ios-macabi",
+        &[
+            ("CMAKE_OSX_ARCHITECTURES", "arm64"),
+            ("CMAKE_OSX_SYSROOT", "macosx"),
+            ("CMAKE_MACOSX_BUNDLE", "OFF"),
+        ],
+    ),
     // tvOS
     (
         "aarch64-apple-tvos",
@@ -315,6 +324,8 @@ fn get_boringssl_cmake_config(config: &Config) -> cmake::Config {
             // Hack for Xcode 10.1.
-            let target_cflag = if config.target_arch == "x86_64" {
+            let target_cflag = if config.target == "aarch64-apple-ios-macabi" {
+                "--target=arm64-apple-ios-macabi"
+            } else if config.target_arch == "x86_64" {
                 "-target x86_64-apple-ios-simulator"
             } else {
                 ""
             };
